frontend/src/App.js

```diff
@@ -1,9 +1,34 @@
-import React from 'react'
+import React from "react";
+import { Provider } from "react-redux";
+import { configureStore, combineReducers } from "@reduxjs/toolkit";
+import { BrowserRouter, Switch, Route } from "react-router-dom";
+
+import Main from "./components/Main";
+import Signin from "./components/Signin";
+import Signup from "./components/Signup";
+
+import user from "./reducers/user";
+import thoughts from "./reducers/thoughts";
+
+
+const reducer = combineReducers({
+  user: user.reducer,
+  thoughts: thoughts.reducer
+ 
+});
+
+const store = configureStore({ reducer });
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <BrowserRouter>
+      <Provider store={store}>
+        <Switch>
+          <Route exact path="/" component={Main} />
+          <Route path="/signin" component={Signin} />
+          <Route path="/signup" component={Signup} />
```

Comment:
both forms look 99% the same, maybe it would be nice to build 1 and pass differences as props?